sdtensor.hpp

```diff
@@ -0,0 +1,115 @@
+#ifndef UTENSOR_SDTENSOR_H
+#define UTENSOR_SDTENSOR_H
+#include "tensor.hpp"
+#include "tensorIdxImporter.hpp"
+template <class T>
+class SDTensor : public Tensor {
+  public:
+    SDTensor(TName _name, std::string filename) : Tensor(_name) {
+        _filename = filename;
+        cursor = 0;
+        dirty = false;
+    }
+
+    SDTensor(std::initializer_list<uint32_t> l, TName _name, std::string file) : Tensor(_name) {
```

Comment:
What exactly is the std::initializer_list for?

Reply:
It is for the conciseness of the syntax